cert/aziot-certd/src/config.rs

```diff
@@ -157,29 +157,64 @@ impl<'de> serde::Deserialize<'de> for Est {
 	}
 }
 
+impl serde::Serialize for Est {
+	fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error> where S: serde::ser::Serializer {
+		let mut inner = EstInner {
```

Comment:
I haven't looked too deeply into where this struct is used, but is there any reason you've got this manually-flattened inner struct vs. using something like `#[serde(flatten)]`?

Reply:
I don't think `EstAuthX509` can be represented even with `flatten`ing.